.github/workflows/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Orion documentation
+    url: https://fiware-orion.readthedocs.io/en/master/
+    about: check the documentation before asking help or reporting a bug.
+  - name: Community Support
+    url: https://stackoverflow.com/questions/ask?tags=fiware-orion+fiware
+    about: Please ask and answer questions in StackOverflow and use the tag fiware-orion.
```

Makes sense this?

```suggestion
    about: If you have  actually a question about functionality rather than a bug report or feature request, please consider to ask using StackOverflow using the tag fiware-orion instead of opening an issue in the Orion repository.
```

(Not sure, because I don't know where this hint message is to be used)